code/src/components.js/ToDoList.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import todos from 'reducers.js/todos';
+import { Buttons } from './AddToDo';
+import { LeftToDo, NewToDoP, ToDoListBox } from './GlobalStyles';
+
+const ToDoList = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    const todosFromLocalStorage = JSON.parse(localStorage.getItem('toDoList'));
+    if (todosFromLocalStorage) {
+      dispatch(todos.actions.setupStore(todosFromLocalStorage))
+    }
+  }, [dispatch])
+  const toDoList = useSelector((store) => store.todos.items);
+  const onDeleteSingleToDoBtnClick = (id) => {
+    dispatch(todos.actions.deleteSingleToDo(id));
+  }
+  const onIsDoneCheckboxToggle = (id) => {
+    dispatch(todos.actions.toggleIfToDoIsDone(id));
+  }
+  return (
+    <section>
+      <section>
+        {toDoList.map((singleToDo) => {
+          return (
+            <li key={singleToDo.id}>
+              <ToDoListBox>
+                <LeftToDo>
+                  <input id={`toDo_with_id${singleToDo.id}`} type="checkbox" value={singleToDo.IsDone} onChange={() => onIsDoneCheckboxToggle(singleToDo.id)} />
```

Comment:
To increase user accessibility and make the page more user-friendly you can connect the label with the checkbox so that the user can click on the text for the checkbox and not just on the actual checkbox. I think you need to wrap the text (that you have in a span maybe?) inside of the label. You might also want to add a focus to the checkbox in CSS so that keyboard users see where they are when tabbing.